src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp

```diff
@@ -660,7 +654,7 @@ void ControllerScriptEngineLegacy::handleScreenFrame(
         emit previewRenderedScreen(screenInfo, screenDebug);
     }
 
-    QByteArray input(std::bit_cast<const char*>(frame.constBits()), frame.sizeInBytes());
+    QByteArray input(reinterpret_cast<const char*>(frame.constBits()), frame.sizeInBytes());
```

Comment:
From what I understand, casting between `const char *` and `const unsigned char *` using `reinterpret_cast` seems to be fine, [stackoverflow confirms that](https://stackoverflow.com/questions/28258136/using-a-pointer-for-casting-char-to-unsigned-char). This would fix the build on some older compilers where `std::bit_cast` isn't available, such as the macOS 11 runners we are building on in CI.

Would appreciate a second pair of eyes checking that this is fine though.